.gitignore

```diff
@@ -14,3 +14,12 @@
 /.vscode
 
 /vendor
+
+examples/cosmos/cosmos
+examples/edgelb/edgelb
+examples/iam/iam
+examples/kubernetes/kubernetes
+examples/marathon/marathon
+examples/metadata/metadata
+examples/metronome/metronome
+examples/secrets/secrets
```

Comment:
what is the reason for these gitignore additions?

Reply:
Don't want to include the compiled binaries in the source tree

Reply:
🤔 what speaks against `go run` ?

Reply:
Just `go build` out of habit really; the binaries still shouldn't be tracked though.

Reply:
I also just use `go run` for the examples, but doesn't hurt to have this in gitignore :+1: